lib/mp-readline/readline.c

```diff
@@ -98,6 +98,72 @@ typedef struct _readline_t {
 
 STATIC readline_t rl;
 
+#if MICROPY_REPL_EMACS_WORDS_MOVE
+STATIC size_t cursor_count_forward_word(size_t cursor_pos, vstr_t *line) {
+    size_t pos = rl.cursor_pos;
+    const char *line_buf = vstr_str(line);
```

Comment:
Move this to where it actually gets used?

Reply:
What do you mean? To move it to the first line where it's used in `unichar_isalnum`?

Reply:
Yes, that's clearer (to me at least)

Reply:
Okay, moved

Reply:
Now after optimizing those function, it's organized differently.